src/components/ComingSoon/ComingSoon.js

```diff
@@ -6,9 +6,9 @@ import logo from '../Home/assets/logo-background.svg'
 
 class ComingSoon extends React.Component {
```

Comment:
We should probably remove this component altogether.

Reply:
Done